src/components/accordion/AccordionItem.tsx

```diff
@@ -97,8 +101,23 @@ export const AccordionItem = ({ title, body }: AccordionItem) => {
         onPress={onItemPress}
       >
         <View style={styles.textContainer}>
-          <View style={{ flexShrink: 1, marginRight: accordionIconMargin }}>
-            <H6 color="black">{title}</H6>
+          <View
+            style={[
+              IOStyles.row,
+              IOStyles.alignCenter,
+              {
+                flexShrink: 1
```

Comment:
The const `accordionIconMargin` has been added to add a margin between the text and the chevron icon. With this rewrite, we should define a new `accordionChevronMargin`: 
```suggestion
                flexShrink: 1,
                marginRight: accordionChevronMargin // to be added
```
The original `accordionIconMargin` should be `12` (not `8`), while the new `accordionChevronMargin` should have the original value, so `8`